archive-editorial.php

```diff
@@ -35,7 +35,7 @@
     the_post();
 ?>
 
-    <?php the_post_thumbnail('slide', array('data-id' => $post->ID, 'class' => 'archive-image')); ?>
+    <?php the_post_thumbnail( 'full', array('data-id' => $post->ID, 'class' => 'archive-image') ); ?>
```

Comment:
ah its inside a col6. so I would set an image size called 'col6' with the width whatever col6 is and no cropping